### DISCUSSION TITLE
Implement LSB functinos for init.d

### DISCUSSION THREAD

**Discussion by arcsector** [sakai135/wsl-vpnkit#113] [Ideas]
LSB Conventions require that scripts support a few different commands. One of those is `status`, and another is `restart`:

>LSB-compliant init scripts need to:
> - provide, at least, the following actions: start, stop, restart, force-reload, and status. All of those, except for status, are required by the [Debian Policy, chapter 9.3.2 Writing the scripts](http://www.debian.org/doc/debian-policy/ch-opersys.html#s9.3.2). "Status" support has been requested into policy...

Based on this [LSB Wiki](https://wiki.debian.org/LSBInitScripts), I think that it would help users to include a `status` function, and maybe even a `restart` function as well. The `status` function would be successful based on whether or not the PID file existed, and the restart function would attempt to stop and then start the kit again.

**Comment by sakai135** [comment]
Thanks, this was implemented in v0.3.5
